SConstruct

```diff
@@ -12,7 +12,7 @@ opts.Add(EnumVariable('platform', "Compilation platform", '', ['', 'windows', 'x
 opts.Add(EnumVariable('p', "Compilation target, alias for 'platform'", '', ['', 'windows', 'x11', 'linux', 'osx']))
 opts.Add(BoolVariable('use_llvm', "Use the LLVM / Clang compiler", 'no'))
 opts.Add(PathVariable('target_path', 'The path where the lib is installed.', 'demo/bin/'))
-opts.Add(PathVariable('target_name', 'The library name.', 'libgitapi', PathVariable.PathAccept))
+opts.Add(PathVariable('target_name', 'The library name.', 'gitplugin', PathVariable.PathAccept))
```

Comment:
The library name is used as `libgitplugin` elsewhere. This should be changed to `libgitplugin` too